File: trilobytelib/src/main/java/com/st/trilobyte/models/PropertyEnumValue.java
package com.st.trilobyte.models;

import java.io.Serializable;

public class PropertyEnumValue implements Serializable {

    private String label;

    private int value;

    public String getLabel() {
        return label;
    }

    public int getValue() {
        return value;
    }

    @Override
    public String toString() {
        return "PropertyEnumValues{" +
                "label='" + label + '\'' +
                ", value=" + value +
                '}';
    }
}

File: st_flow_demo/src/main/java/com/st/flow_demo/models/PropertyEnumValue.kt
package com.st.flow_demo.models
import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class PropertyEnumValue(
    @SerialName(value = "label")
    var label: String,
    @SerialName(value = "value")
    var value: Int
)